runtime/lua/vim/_defaults.lua

```diff
@@ -928,14 +928,17 @@ do
 
   vim.api.nvim_create_autocmd('VimEnter', {
     group = vim.api.nvim_create_augroup('nvim.find_exrc', {}),
-    desc = 'Find project-local configuration',
+    desc = 'Find exrc files in parent directories',
     callback = function()
-      if vim.o.exrc then
```

Comment:
We should leave the top-level check in as well, that way we don't waste time doing a traversal if it's not wanted

Reply:
Yeah I was in doubt about that. Thought the traversal wasn't that expensive (bc it's upwards), but it's still a nice optimization.

Reply:
We can change it to

```lua
if not vim.o.exrc then
  return true
end
```

so that the function body isn't indented.